code/modules/mob/living/simple_animal/abnormality/aleph/white_night.dm

```diff
@@ -3,8 +3,8 @@ GLOBAL_LIST_EMPTY(apostles)
 /mob/living/simple_animal/hostile/abnormality/white_night
 	name = "White night"
 	desc = "The heavens' wrath. Say your prayers, heretic, the day has come."
-	health = 12000
-	maxHealth = 12000
+	health = 15000
```

Comment:
Egor, he already had 3 times the amount of HP as every other Aleph, he doesn't need 4x.
This is too much.

Reply:
This is a static target that does not move. It is very much needed.